projects/1.20-fabric/assets/neepmeat/neepmeat/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+  "block.neepmeat.metal_barrel": "金属桶",
+  "block.neepmeat.content_detector": "物品探测器",
+  "block.neepmeat.content_detector.lore_0": "探测面向容器中的特定物品。可用漏斗放入和去除过滤物品。",
+  "block.neepmeat.solidity_detector": "固体探测器",
+  "block.neepmeat.solidity_detector.lore_0": "若面向方块不是空气或流体，发出红石信号。",
+
+  "block.neepmeat.bottler": "灌装机",
+  "block.neepmeat.bottler.lore_0": "将流体注入物品。",
+  "block.neepmeat.basic_tank": "基础储罐",
+  "block.neepmeat.basic_tank.lore_0": "可装下8桶流体。",
+  "block.neepmeat.basic_tank.lore_1": "可纵向堆叠。",
+  "block.neepmeat.basic_tank.filled": "基础储罐（%s）",
+  "block.neepmeat.basic_glass_tank": "视窗储罐",
+  "block.neepmeat.basic_glass_tank.lore_0": "可装下8桶流体。",
+  "block.neepmeat.basic_glass_tank.lore_1": "可纵向堆叠。",
+  "block.neepmeat.basic_glass_tank.filled": "视窗储罐（%s）",
+  "block.neepmeat.advanced_tank": "高级储罐",
+  "block.neepmeat.advanced_tank.lore_0": "可装下16桶流体。",
+  "block.neepmeat.advanced_tank.lore_1": "可纵向堆叠。",
+  "block.neepmeat.advanced_tank.filled": "高级储罐（%s）",
+
+  "block.neepmeat.multi_tank": "多容储罐",
+
+  "block.neepmeat.flex_tank": "变容储罐",
+  "block.neepmeat.flex_tank.lore_0": "方块越多，容量越大。",
+  "block.neepmeat.flex_tank.lore_1": "每个方块可装下8桶流体。",
+  "block.neepmeat.advanced_flex_tank": "高级变容储罐",
+  "block.neepmeat.advanced_flex_tank.lore_0": "方块越多，容量越大。",
+  "block.neepmeat.advanced_flex_tank.lore_1": "每个方块可装下16桶流体。",
+
+  "block.neepmeat.fluid_buffer": "流体缓冲池",
+  "block.neepmeat.fluid_buffer.lore_0": "能容纳1桶流体。",
+  "block.neepmeat.fluid_drain": "流体漏集池",
+  "block.neepmeat.fluid_drain.lore_0": "收集世界中的流体。",
+  "block.neepmeat.fluid_interface": "流体接口",
+  "block.neepmeat.fluid_interface.lore_0": "用作所面对方块的代理。",
+
+  "block.neepmeat.fluid_gauge": "流体存量计",
+  "block.neepmeat.fluid_gauge.lore_0": "根据储罐的填满程度发出红石信号。",
+  "block.neepmeat.item_gauge": "物品存量计",
+  "block.neepmeat.item_gauge.lore_0": "根据容器的填满程度发出红石信号。",
+
+  "block.neepmeat.converter": "流体充能器",
+  "block.neepmeat.transducer": "燃液换能器",
+  "block.neepmeat.power_flower_seeds": "能量花种子",
+  "block.neepmeat.power_flower_seeds.lore_0": "可种在泥土或草方块里。会生长成壮观的有机产能器。",
+  "block.neepmeat.power_flower_growth": "能量花增生体",
+  "block.neepmeat.power_flower_growth.lore_0": "能量花结构的基础。\n上方与空气接触的增生体会特化为光合器官，其余增生体会增加代谢率。\n施加骨粉或肉时会扩散。\n可用精准采集取下。",
+  "block.neepmeat.power_flower_fluid_port": "能量花流体口",
+  "block.neepmeat.power_flower_fluid_port.lore_0": "能将水和液态食物吸入能量花。",
+  "block.neepmeat.power_flower_controller": "能量花核心",
+  "block.neepmeat.power_flower_controller.lore_0": "能量花产出的缥缈燃液自此流出。\n需向能量花流体口送入水。\n可直接破坏取下。",
+  "block.neepmeat.fluid_exciter": "心脏机",
+  "block.neepmeat.fluid_exciter.lore_0": "下方接受缥缈燃液时，上部即向脉管网络供能。效率是标准马达的150%。",
+
+  "block.neepmeat.verrucous_stone": "疣斑石",
+  "block.neepmeat.verrucous_stone_port": "疣斑石口",
+
+  "block.neepmeat.item_buffer": "展示台",
+  "block.neepmeat.item_buffer.lore_0": "潜行时点击以调整容量。",
+  "screen.neepmeat.display_plate.capacity": "输入新容量",
+
+  "block.neepmeat.item_duct": "漏斗延长管",
+
+  "block.neepmeat.rusty_panel": "锈蚀金属横板",

```

```suggestion
  "block.neepmeat.rusty_panel": "锈蚀横板",
```
有的没Metal没加金属，有的加了金属，需要统一